Interfaces/Model/UiPageInterface.php

```diff
@@ -331,24 +345,51 @@ public function setContents($contents);
     public function copy($page_alias = null, $page_uid = null);
     
     /**
-     * Compares two pages by their UIDs and returns true if they are equal.
+     * Compares two pages by their UIDs, aliases and CMS-IDs and returns
+     * true if they are equal.
      * 
      * If the passed page replaces this page this function also returns true.
      * 
-     * @param UiPageInterface $page
+     * @param UiPageInterface|string $page_or_id_or_alias
      * @return boolean
      */
-    public function is(UiPageInterface $page);
+    public function is($page_or_id_or_alias);
     
     /**
-     * Compares two pages by their UIDs and returns true if they are equal.
+     * Compares two pages by their UIDs, aliases and CMS-IDs and returns
+     * true if they are equal.
      * 
      * If the passed page replaces this page this function returns false.
      * 
+     * @param UiPageInterface|string $page_or_id_or_alias
+     * @return boolean
+     */
+    public function isExactly($page_or_id_or_alias);
+    
+    /**
+     * Compares two pages by their attributes ignoring their positions in the
+     * menutree.
+     * 
      * @param UiPageInterface $page
+     * @param string[] $skip_property_names
      * @return boolean
      */
-    public function isExactly(UiPageInterface $page);
+    public function equals(UiPageInterface $page, $skip_property_names);
```

Comment:
Statt $skip_property_names lieber $ignor_properties und bitte auch in der Beschreibung erwähnen.